test/unit/testUtil.js

```diff
@@ -32,6 +32,9 @@ define('testUtil', ['bluebird', 'json!/test/testConfig.json'], (Promise, TestCon
                 'Missing an auth token. Please enter one (or null to skip those tests) in test/unit/testConfig.json'
             );
         }
+        if (TestConfig.token === null) {
+            return new Promise((resolve) => resolve(null));
+        }
```

Comment:
The instructions above mention having the token set to null, but there wasn't any code to handle a null token... until now!